platformio.ini

```diff
@@ -12,7 +12,8 @@
 default_envs = rack32-debug
 
 [firmware]
-name = \"OXRS-SHA-StateIO-ESP32-FW\"
+name_room8266 = \"OXRS-SHA-StateIO-ESP8266-FW\"
```

Comment:
Maybe these should be `name_esp8266` and `name_esp32`?

Reply:
I tried to reference the name to the env, but referencing to the MCU could be done as well.
Let me know your preference.

Reply:
If we added support for the ESP32 LillyGO then it would use the same name_esp32 wouldn't it? 

Wouldn't make sense for that to be name_rack32 I don't think.

Reply:
good point !